tests/flow/test_maxflow.py

```
@@ -1313,3 +1313,32 @@ def test_36_max_flow_default_capacity_double(self):
             YIELD maxFlow
         """)
         self.env.assertAlmostEqual(result.result_set[0][0], 4.5, 5)
+
+    def test_37_no_edges_of_relationship_type(self):
+        """When the relationship type exists but no edges survive the label
+        filter, maxFlow should return one row with maxFlow=0 and empty arrays."""
+        # Create a PIPE edge (registers the rel-type in the schema) between
+        # nodes labelled 'X', then query between nodes labelled 'Y' where no
+        # PIPE edges exist – Build_Matrix produces an empty matrix.
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)